src/components/route-button-container.tsx

```diff
@@ -12,38 +19,39 @@ const styles = StyleSheet.create({
 	},
 });
 
-interface Props {
-	/** Function to start and stop route tracking */
-	changeTracking: () => void;
-	/** Function to toggle show route state true or false */
-	changeShowRoute: () => void;
-	/** State to determine whether route is visible or not */
-	showRoute: boolean;
-	/** State to determine whether route tracking has started or not */
-	isTracking: boolean;
-}
-
 /**
  * Route button container component which contains buttons to
  * start / end route tracking and to toggle route visibility on / off.
  * Just preliminary styling and location on a screen.
  * @returns a tree of React elements
```

Comment:
We probably don't need explicit `@returns` documentation for components, since they will all by definition return a react subtree 😅 I like the spirit, though!

Reply:
Removed :smiley: